catalog/installWebSocket.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# use the command line interface to install Weather.com package.
+#
+: ${WHISK_SYSTEM_AUTH:?"WHISK_SYSTEM_AUTH must be set and non-empty"}
+AUTH_KEY=$WHISK_SYSTEM_AUTH
+
+SCRIPTDIR="$(cd $(dirname "$0")/ && pwd)"
+CATALOG_HOME=$SCRIPTDIR
+source "$CATALOG_HOME/util.sh"
+
+echo Installing WebSocket package.
+
+createPackage websocket \
+    -a description "Utilities for communicating with WebSockets"
+
+waitForAll
+
```

Comment:
1. field-level descriptions, e.g. what does the uri or payload field mean?
2. i would guess that we should mark the uri field as bindTime. which means that the package-level metadata needs a "parameters" annotation `[ { "key": "parameters": value: [{ "name": "uri", bindTime: true, required: true } }]`

3. we don't have an established schema for this yet, but the package also needs these annotations
  * service icon
  * branding text
  * branding icon (optional, e.g. if the service provider has a Cloudant<TM> wider branding icon)
  * link to service provider's home page
  * link to service instantiation broker